src/gui/meson.build

```diff
@@ -4,6 +4,7 @@ libgui_sources = files(
     'sdl_mapper.cpp',
     'sdlmain.cpp',
     'shader_manager.cpp',
+    'title_bar.cpp',
```

Comment:
I'm fine with "titlebar" (one word) which you seem to be using consistently everywhere. But then the file should be called `titlebar.cpp`.

Reply:
Ok, file name changed.